cliquet/tests/resource/test_events.py

```diff
@@ -0,0 +1,34 @@
+import webtest
+from pyramid.config import Configurator
+
+from cliquet.events import CollectionChanged
+from cliquet.tests.testapp import main as testapp
+from cliquet.tests.support import unittest, BaseWebTest, get_request_class
+
+
+class EventsTest(BaseWebTest, unittest.TestCase):
+
+    def get_test_app(self, settings=None):
+        settings = self.get_app_settings(settings)
+        self.config = Configurator(settings=settings)
+        app = testapp(config=self.config)
+        app = webtest.TestApp(app)
+        app.RequestClass = get_request_class(self.api_prefix)
+        return app
+
+    def test_events_triggered(self):
+        # let's subcribe to the CollectionChanged event
+        events = []
+
+        def listener(event):
+            events.append(event)
+
+        self.config.add_subscriber(listener, CollectionChanged)
+        self.config.commit()
+
+        # creating a record should trigger an event
+        body = {'data': {'name': 'de Paris'}}
+        self.app.get(self.collection_url, headers=self.headers, status=200)
```

Comment:
I know this is tempting to do it that way :)  But the assertion is hidden.

Please use setUp to plug your listener and add specific tests like `test_does_not_trigger_on_read` ... `triggers_on_write` etc.